<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialogProgress</class>
 <widget class="QDialog" name="QDialogProgress">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDialogProgress</string>
  </property>
  <property name="windowOpacity">
   <double>0.500000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="styleSheet">
      <string notr="true">#widgetMain{
	border-radius: 25px;
	background-color: rgba(0, 0, 0, 160);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="topMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetTop" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QProgressIndicator" name="widgetProgress" native="true">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>120</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>120</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="labelProgress">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Microsoft YaHei&quot;;</string>
              </property>
              <property name="text">
               <string>100%</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMsg">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei&quot;;</string>
        </property>
        <property name="text">
         <string>Loading</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QProgressIndicator</class>
   <extends>QWidget</extends>
   <header>QProgressIndicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
